Dockerfile

```diff
@@ -9,16 +9,20 @@ ENV LANG C.UTF-8
 
 # https://support.circleci.com/hc/en-us/articles/360045268074-Build-Fails-with-Too-long-with-no-output-exceeded-10m0s-context-deadline-exceeded-
 ENV PYTHONUNBUFFERED=1
+ENV NODE_VERSION=node_16.x
+
 
 RUN \
   # Install system dependencies
   apt-get update \
   && apt-get install -y --no-install-recommends \
+      apt-transport-https \
```

Comment:
Since these commands are shared by multiple Dockerfiles, I wonder if it would make sense to put them in a script and run that instead?

Reply:
it shouldn't be shared for long, we are working on moving everything over to the buster image, and then this one can go away.